drf_spectacular/utils.py

```diff
@@ -494,10 +494,17 @@ def decorator(view):
                 )
                 continue
 
-            # the context of derived methods must not be altered, as it belongs to the
-            # other view. create a new context so the schema can be safely stored in the
-            # wrapped_method. view methods that are not derived can be safely altered.
-            method_decorator(isolate_view_method(view, method_name))
+            if hasattr(method_decorator, '__iter__'):
+                for md in method_decorator:
```

Comment:
1. could you name `md` something more expressive like `sub_method_decorator`
2. could deduplicate the comment and just move it above line 497